bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -37,10 +37,11 @@
       {
         "uiName": "Reader schema version",
         "fieldName": "readerSchemaVersion",
-        "isOptional": true,
+        "isOptional": false,
```

Comment:
@satishd 
Two scenarios:
1. If user imports the topology and directly tries to deploy, that will still work since all the streams/edges and everything was properly linked (unless on runtime backend does a validation on every component field whether a mandatory one is missing or not)

2. If user imports and opens the component (in this case Kafka source), the missing field will be blank and if the user doesn't add value for that field, UI won't allow to save & keeps showing `Required Field` error.